docs/ko/week11/11-1.md

```diff
@@ -0,0 +1,1010 @@
+﻿---
+
```

Comment:
Remove empty lines here and below.

Reply:
I don't know why those useless empty lines were created. I corrected all those things.